setup.py

```diff
@@ -105,6 +105,19 @@ def initialize_options(self) -> None:
     #
     # https://github.com/googleapis/google-auth-library-python/issues/190
     'google-api-python-client>=1.8.0,<1.9.0',
+    #
+    # For some reason this issue started happening consistently around 2020-10:
+    #
+    #  https://app.circleci.com/pipelines/github/bluelabsio/records-mover/1134/workflows/267ca651-def1-4f60-bdfb-0f857a9e0c60/jobs/9770
+    #
+    # This seems to be a clue:
+    #
+    #  https://github.com/pypa/pip/issues/8407
```

Comment:
Praying for the new resolution stuff to be out and stable soon-- although i'm sure it's going to break all sorts of back/forwards compatibility things, sigh.